sphinxcontrib/qthelp/__init__.py

```diff
@@ -102,7 +107,7 @@ def build_qhp(self, outdir: str, outname: str) -> None:
 
         for indexname, indexcls, content, collapse in self.domain_indices:
             item = section_template % {'title': indexcls.localname,
-                                       'ref': '%s.html' % indexname}
+                                       'ref': ''.join((indexname, self.out_suffix))}
```

Comment:
It is much simple and understandable to concat them by `+`. I feel it is a bit strange to use `join()` here, but another one uses `%` formatting.